wftests/warrior_tests/testcases/common_actions_tests/test_slurp_json.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" ?>
+<Testcase>
```

Comment:
Negative/Failing scenarios are not being tested in this testcase. Please add those.

Reply:
sure. will add those.

Reply:
Please add comprehensive negative testing scenarios - you can also create a new testcase containing those:

Examples will be:
1. Incorrect file path
2. Missing filepath
3. Misspelled bool_store_all value
4. False bool_store_all value
5. Slurp file with invalid json
6. Overwriting an existing key in datarepository
and other scenarios and valid combinations thereof.

Reply:
done. added testcase

Reply:
done